tests/system/providers/google/cloud/gcs/example_gcs_to_bigquery_async.py

```diff
@@ -81,6 +81,20 @@
         max_id_key=MAX_ID_DATE,
         deferrable=True,
     )
+
```

Comment:
Can we add a test with non-default values of `field_delimiter`, `quote_character` parameters?

Reply:
Done